javascript/packages/fury/lib/fury.ts

```diff
@@ -44,6 +48,8 @@ export default class {
     this.referenceResolver = new ReferenceResolver(this.binaryReader);
     this.classResolver.init(this);
     this.anySerializer = new AnySerializer(this);
+    this.typeMeta = new TypeMeta(BigInt(0),[]);
```

Comment:
`from_bytes` in TypeMeta is a static method, if you want to call it, you should replace by `this.typeMeta = TypeMeta`